### ISSUE TITLE
Peg leg surgery consumes the entire stack of wood.

### ISSUE THREAD

**Issue by Sooose** [d3athrow/vgstation13#16097]
#### Description of issue
Peg legs only require one plank of wood to make, however the surgery step consumes the entire stack of wood.

#### Difference between expected and actual behavior
Expected the rest of the wood not to vanish into thin air.

#### Steps to reproduce 
Attempt peg leg surgery with a stack of more than one plank of wood in hand.

**Comment by Cyadox** [comment]
~~~I was just working on peg legs, I'll take a peek and see whats up.~~~
Code looks sound, removes one from the stack when using it for the peg limb.

```
else if(istype(I, /obj/item/stack/sheet/wood))
		var/obj/item/stack/sheet/wood/peg = I
		if(peg.use(1))
			src.peggify()
```

And /proc/use:
```
/obj/item/stack/proc/use(var/amount)
	ASSERT(isnum(src.amount))
	if(src.amount>=amount)
		src.amount-=amount
		update_materials()
	else
		return 0
	. = 1
```

I did find during testing the peg leg removal bug that after I used a plank, if I had any remaining it would work as intended until I dropped or threw it, in which case the entire stack would vanish. I thought it was just how I spawned it, but seems that it could be related.
